transaction-pool/src/tests/mod.rs

```diff
@@ -210,21 +210,24 @@ fn should_construct_pending() {
 
 	let tx0 = txq.import(b.tx().nonce(0).gas_price(5).new()).unwrap();
 	let tx1 = txq.import(b.tx().nonce(1).gas_price(5).new()).unwrap();
+
+	let tx9 = txq.import(b.tx().sender(2).nonce(0).new()).unwrap();
```

Comment:
Why did you need to change the order of imported of txs?

Reply:
Cause it was actually affected by the change - previously `tx9` was returned as one of the first pending transactions, because it was inserted last (with gas_price=0 as the rest).

I didn't change the `assert`, but rather changed the ordering, to make sure the tests are passing.

Reply:
Thanks :+1: Just thought ordering wasn't affected on all of them but I probably overlooked the test.